cypress/integration/twilio-video.spec.js

```diff
@@ -31,28 +32,40 @@ const caseRef = uuid();
                 });
 
       beforeEach(() => {  
-                   cy.visit(`${loginUrlPath}/login/`);
+                  cy.visit(`${baseUrl}/auth/logout`);
+                   cy.visit(`${loginUrlPath}/login?UserIdentifier=&Language=en-us&CaseReference=&Password=`);
                 });
-     
-      it('should fill login form and get error of "no active hearing for the case number"', () => {
 
-            cy.fillConferenceLoginPage('abfhg','123$567','1313');
+          it('should fill login form and get error of "The username/password is incorrect."', () => {
+            let userName = 'abfhg';
```

Comment:
Will it work in all environments? Shall we prepare the test data and create the user/conference...?

Reply:
I would suggest to use random and named parameters
Next time I suggest we'll use TypeScript